src/blocks/Slider/Slider.tsx

```diff
@@ -224,24 +224,28 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
 
         return (
             slidesCountByBreakpoint > 1 && (
-                <div
-                    className={b('bar')}
-                    style={{
-                        left: barPosition,
-                        width: barWidth,
-                    }}
-                />
+                <li>
```

Comment:
why slider by is <li>?

Reply:
validator.w3.org informs us that 'ul' tag should have only 'li' tag as a child